src/gateways/createArchiveLoader.ts

```diff
@@ -58,14 +60,16 @@ export default (url: string) => {
   return {
     book: (bookId: string, bookVersion?: string) => {
       const bookRef = bookVersion ? `${stripIdVersion(bookId)}@${bookVersion}` : stripIdVersion(bookId);
+      const bookAndPageRef = (pageId: string) => `${bookRef}:${pageId}`;
 
       return {
         cached: () => bookCache.get(bookRef),
         load: () => bookLoader(bookRef),
 
         page: (pageId: string) => ({
```

Comment:
give this function a body instead of the return object shorthand and you can define your `bookAndPageRef` at the top and not have the extra function

Reply:
I don't understand what you mean about _give it a body_

Reply:
right now the function is like
```
(pageId: string) => ({
})
```
returning an object literal, you can change it to
```
(pageId: string) => {
  const theThing = 'asfd';
  return {
  };
}
```